cms/djangoapps/contentstore/views/tests/test_block.py

```diff
@@ -312,12 +312,12 @@ def test_split_test(self):
         resp = self.create_xblock(
             parent_usage_key=split_test_usage_key,
             category="html",
-            boilerplate="zooming_image.yaml",
+            boilerplate="latex_html.yaml",
         )
         self.assertEqual(resp.status_code, 200)
         html, __ = self._get_container_preview(split_test_usage_key)
         self.assertIn("Announcement", html)
-        self.assertIn("Zooming", html)
+        self.assertIn("LaTeX", html)
```

Comment:
Where did this come from?

Reply:
It comes from [this PR](https://github.com/openedx/edx-platform/pull/31435) that was merged before #36012

Reply:
The test suite for these templates just randomly included some of them (and not others).  When removing the tool originally, I just added another random one. 🤷🏼